File: discord/nim/bot.nim
# nimble install dimscord

import dimscord, asyncdispatch, times, options

let discord = newDiscordClient("TOKEN")


proc messageCreate(s: Shard, m: Message) {.event(discord).} =
    if m.author.bot: return
    if m.content == "Fuck": 
    for line in lines "../../countries.txt":
    msg = await discord.sendMessage(m.channel, "Fuck" line)

waitFor discord.startSession()

File: discord/lua/bot.lua
local discordia = require('discordia')

local client = discordia.Client()


client:on('messageCreate', function(message, filename, fp, io)
	if message.content == 'Fuck' then
        filename = "countries.txt"
        fp = io.open( filename, "r" )
           for line in fp:lines() do
            message.channel:sendMessage(line)
         end
      end
  end)


client:run('TOKEN')
